### PR TITLE
iOS and Android 2.0.0 changelogs

### PR DESCRIPTION
**Pull request by valllllll2000**
#### Steps to test

- Go to the website project.
- Run `env DOCS_BRANCH=this-pr-branch yarn docs` to download the docs files from this PR.
- Run `yarn start` and go to `/docs` to check the changes.

Updated the releases section for the Android SDK, task:  https://fidelapi.atlassian.net/browse/CLET-57 and https://fidelapi.atlassian.net/browse/CLET-59

### CODE REVIEW

**Review by diazproduction — APPROVED** [review]

**Review by MariaJoaoMiraPaulo — DISMISSED** [review]

File: select/sdks/ios/releases.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# iOS SDK v2 Releases
+
+## 2.0.0
+- Removed card scanning so `Fidel.autoScan` has been removed.
```
Comment (cornelfd) [inline select/sdks/ios/releases.md:4]: We should also add that we dropped support for Objective-C.

Comment (cornelfd) [reply select/sdks/ios/releases.md:4]: @valllllll2000 please make sure to update the Android changelog as well with the changes from the doc I linked above.

File: stream/sdks/ios/releases.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # iOS SDK v2 Releases
 
+## 2.0.0
+
+- Stable release, stream is no longer in beta
```
[the remark this answers is not in the document]
Comment (valllllll2000) [reply stream/sdks/ios/releases.md]: Fixed

**Review by MariaJoaoMiraPaulo — APPROVED** [review]

**Review by cornelfd — APPROVED** [review]

File: select/sdks/ios/releases.md (hunk 1 above)
Comment (valllllll2000) [reply select/sdks/ios/releases.md:4]: Done

**Pull request merged**